libcloud/test/compute/test_openstack.py

```diff
@@ -2674,6 +2680,11 @@ def _v2_1337_v2_0_routers_f8a44de0_fc8e_45df_93c7_f79bf3b01c95_remove_router_int
         if method == 'PUT':
             body = self.fixtures.load('_v2_0__router_interface.json')
             return (httplib.OK, body, self.json_content_headers, httplib.responses[httplib.OK])
+
+    def _v2_1337_servers_1000_action(self, method, url, body, headers):
+        if method != 'POST' or body != '{"removeSecurityGroup": {"name": "sgname"}}':
```

Comment:
Good test case :+1: 

We should use such assertions in more places (aka asserting also that the body is correct and not just the path / URL).